plugins/obs-filters/CMakeLists.txt

```diff
@@ -1,111 +1,141 @@
 project(obs-filters)
 
-option(DISABLE_SPEEXDSP "Disable building of the SpeexDSP-based Noise Suppression filter" OFF)
+option(ENABLE_SPEEXDSP
+       "Enable building with SpeexDSP-based noise suppression filter" ON)
+option(ENABLE_RNNOISE "Enable building with RNNoise noise supression filter" ON)
 
-if(DISABLE_SPEEXDSP)
-	message(STATUS "SpeexDSP support disabled")
-	set(LIBSPEEXDSP_FOUND FALSE)
-else()
-	find_package(Libspeexdsp)
-	if(LIBSPEEXDSP_FOUND)
-		add_definitions(-DLIBSPEEXDSP_ENABLED)
-	endif()
+if(OS_WINDOWS)
+  option(
+    ENABLE_NVAVX
```

Comment:
The old flag is `DISABLE_NVAFX`.  The second half is the part I'm pointing out.

Reply:
Fixed.